tests/unittests/HabanaBackendTest.cpp

```diff
@@ -1711,3 +1711,52 @@ TEST_F(HabanaBackendTest, DISABLED_BatchedGather) {
   EXPECT_FLOAT_EQ(H.at({2, 0}), 4.5);
   EXPECT_FLOAT_EQ(H.at({2, 1}), 1.2);
 }
+
+TEST_F(HabanaBackendTest, DISABLED_BatchedGatherEx) {
+  unsigned M = 1000;
+  unsigned N = 1;
+
+  // Fill out the array with random data
+  std::vector<float> input_data;
```

Comment:
could you fix the naming here. The convention is to use: inputData, InputIds etc.